Source/SwiftLintCore/Rules/SuperfluousDisableCommandRule.swift

```diff
@@ -1,7 +1,7 @@
 package struct SuperfluousDisableCommandRule: SourceKitFreeRule {
     package var configuration = SeverityConfiguration<Self>(.warning)
 
-    package init() {}
+    package init() { /* Make initializer as accessible as its type. */ }
```

Comment:
Maybe here too - if an `init` has any accessibility modifier apart from `internal`, then there is a good reason it might be empty.